tests/scattering1d/test_torch_scattering1d.py

```diff
@@ -436,7 +436,19 @@ def test_Q(device, backend):
     length = 1024
     shape = (length,)
 
+    # test different cases for Q
+    sc_int = Scattering1D(J, shape, Q=(8, ), backend=backend, frontend='torch').to(device)
+
+    sc_tuple = Scattering1D(J, shape, Q=(8, 1), backend=backend, frontend='torch').to(device)
+    
+    assert sc_int.Q == sc_tuple.Q
```

Comment:
let's also check that the outputs of these two `Scattering1D` objects are the same?

Reply:
The outputs of the `Scattering1D` for `Q = (8, )` and `Q = (8, 1)` are checked via `assert sc_int.Q == sc_tuple.Q` for each frontend. I did not get `check that for a dummy input Scattering1D returns the same thing for Q = (8, ) and Q = (8, 1)`. Could you please explain a bit what you mean by `dummy input`? Thanks! @cyrusvahidi

Reply:
look at how `test _T` is written:


```python
Sg0 = scattering0(x)
Sg1 = scattering1(x)
assert torch.allclose(Sg0, Sx0)
assert Sg1.shape == (Sg0.shape[0], Sg0.shape[1], Sg0.shape[2]*2**(sigma_low_scale_factor))
```

here, the "dummy input" is `x`which is filled with zeros. What matters in this test is not the values but the shapes.

Reply:
I see. Thanks a lot! I will do it quickly.